test/jest-puppeteer.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
 		devtools: process.env.HEADLESS === 'false'
 	},
 	server: {
-		command: 'PORT=4444 npm run server'
+		command: 'PORT=4444 npm run server',
+		port: 4444
```

Comment:
heh - I had this locally and assumed it wasn't doing anything. My bad!

Reply:
No worries, I just which the `jest-puppeteer` docs would mention that this is required :man_shrugging: